pkgs/development/libraries/qt-4.x/4.8/default.nix

```diff
@@ -139,37 +139,40 @@ stdenv.mkDerivation rec {
 
   prefixKey = "-prefix ";
 
-  ${if stdenv.hostPlatform == stdenv.buildPlatform then null else "configurePlatforms"} = [];
+  configurePlatforms = [];
   configureFlags = let
+    mk = cond: name: "-${lib.optionalString cond "no-"}${name}";
```

Comment:
This will prepend `no-` when `cond` is true, which is the opposite of the intended effect. We can see in the [qt48 log]](https://hydra.nixos.org/build/79370010/log) that it's building with eg. demos support, but not webkit support. cc @Ericson2314

Reply:
Same thing for qt3.